internal/cmd/operator-sdk/generate/packagemanifests/packagemanifests.go

```diff
@@ -35,8 +35,15 @@ towards using bundles by default. Run 'operator-sdk generate bundle -h' for more
 
 Running 'generate packagemanifests' is the first step to publishing your operator to a catalog and/or deploying
 it with OLM. This command generates a set of manifests in a versioned directory and a package manifest file for
-your operator. Typically one would run 'generate kustomize manifests' first to (re)generate kustomize bases
-consumed by this command.
+your operator. A ClusterServiceVersion manifest will be generated from the set of manifests passed to this command
```

Comment:
this is a mouthful of a sentence and I don't even really understand what it's trying to say. is the existing base the 'manifests passed to this command'? Is the 'from scratch" describing the "'<kustomize-dir>/bases/<package-name>.clusterserviceversion.yaml'," ? what exactly does "from scratch mean"? As if I had just run the 'generate kustomize manifests' command?

Reply:
It overruns a bit I agree.